core/services/vrf/estimate/estimate.go

```diff
@@ -0,0 +1,27 @@
+package estimate
+
+import "math/big"
+
+// GasProofVerification is an upper limit on the gas used for verifying the VRF proof on-chain.
+// It can be used to estimate the amount of LINK needed to fulfill a request.
+const GasProofVerification uint32 = 200_000
+
+// JuelsNeeded estimates the amount of link needed to fulfill a request
+// given the callback gas limit, the gas price, and the wei per unit link.
+func JuelsNeeded(callbackGasLimit uint32, maxGasPriceWei, weiPerUnitLink *big.Int) *big.Int {
```

Comment:
Or maybe having its own package is making it more awkward? Could be `vrf.EstimateFeeJuels`?

Reply:
I don't mind it being FeeJuels. The reason I moved it to its own package is because I was getting import cycles when trying to test the function in listener_v2_test.go for some reason 🤔. I actually had it as a private helper in listener_v2.go initially.

Reply:
OK nbd. Did you try writing the test from `package vrf_test`? That usually enables working around the cycle, and then you could limit the export to just a test helper if you wanted to.

Reply:
Oh you're absolutely right, that should work! Will change it up now.